builddefs/converters.mk

```diff
@@ -29,6 +29,7 @@ ifneq ($(CONVERT_TO),)
 
     # Configure any defaults
     OPT_DEFS += -DCONVERT_TO_$(strip $(shell echo $(CONVERT_TO) | tr '[:lower:]' '[:upper:]'))
+    OPT_DEFS += -DTARGET_CONVERT=\"$(strip $(CONVERT_TO))\"
```

Comment:
Maybe `TARGET_CONVERT` -> `CONVERTER_TARGET`?

Reply:
Sure, I don't see why not :D